.circleci/config.yml

```diff
@@ -23,6 +23,15 @@ references:
       CONFIG_ENV: release
       DETECT_CHROMEDRIVER_VERSION: 'false'
       NODE_ARGS: --max_old_space_size=3072
+  desktop_machine_executor_defaults: &desktop_machine_executor_defaults
+    working_directory: ~/wp-calypso
+    # Use Linux machine executor, NOT docker image for higher available RAM (7.5GB)
+    machine:
+      image: ubuntu-1604:201903-01
+    resource_class: medium
```

Comment:
Per the docs, there are other resource classes available (e.g. `large` with 15GB of RAM. However no other resource classes are available with our current plan, and Circle will kick the job back to a "medium" instance even if a larger size is requested. 🤷